src/csvcubed/models/cube/qb/components/codelist.py

```diff
@@ -107,7 +136,7 @@ class NewQbCodeList(QbCodeList, ArbitraryRdf, Generic[TNewQbConcept]):
     uri_style: Optional[URIStyle] = None
 
     @validator("concepts")
-    def _ensure_no_use_of_reserved_keywords(
+    def pydantic_ensure_no_use_of_reserved_keywords(
```

Comment:
Can you keep this as a privately named function? 

Similarly for `pydantic_validate_concepts_non_conflicting`